dataprovider-retrofit/src/main/java/ch/srg/dataProvider/integrationlayer/request/parameters/IlTransmission.kt

```diff
@@ -7,4 +7,4 @@ import ch.srg.dataProvider.integrationlayer.data.Transmission
  * <p>
  * License information is available from the LICENSE file.
  */
-class ILTransmission(transmission: Transmission) : IlParam(transmission.toString().lowercase())
\ No newline at end of file
+class ILTransmission(transmission: Transmission) : IlParam(transmission.toString().lowercase())
```

Comment:
⚠️ **[detekt]** <detekt.MatchingDeclarationName> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The file name 'IlTransmission' does not match the name of the single top-level declaration 'ILTransmission'.